triton-server/src/download/schema.sql

```diff
@@ -19,10 +19,9 @@ create table requests (
 
     should_delete tinyint not null default 0,
 
-    unique(type, dataset_id)
 );
 
-create index idx_dataset_request on requests(dataset_id);
+create unique index idx_dataset_request on requests(dataset_id);
```

Comment:
That's interesting. Is that a valid syntax?

Reply:
sqlite index do not enforce uniqueness unless you specify them as unique. Since we want that index i removed the unique constraint that was doing the same.